src/main/java/org/reaktivity/command/log/internal/LogStreamsCommand.java

```diff
@@ -67,6 +73,8 @@
         this.affinity = affinity;
         this.position = position;
         this.out = out;
+        this.hasExtensionType = extensionTypes == null ? t -> true : Arrays.asList(extensionTypes)::contains;
+        this.hasFrameType = frameTypes == null ? t -> true : Arrays.asList(frameTypes)::contains;
```

Comment:
Please move these predicates outside constructor, to call site where `String[]` type is forced by command line parser, and pass predicates into constructor instead.